spec/presenters/vendor_api/single_application_presenter_spec.rb

```diff
@@ -585,4 +585,12 @@
       expect(presenter.as_json[:attributes][:recruited_at]).to be_present
     end
   end
+
+  describe 'attributes.status' do
+    it 'returns awaiting_provider_decision when status is interviewing' do
+      application_choice = build_stubbed(:application_choice, status: :interviewing)
```

Comment:
It's probably worth adding a trait that sets up a realistic interviewing application with interviews etc.

Reply:
Oh – I see that's in the next PR 👍